include "parser.thrift"
namespace java com.codingstory.polaris.typedb

struct TTypeData {
    1: parser.TClassType classType;
}
